src/styles/vendor/fullcalendar.css

```diff
@@ -62,42 +57,38 @@
   @apply bg-fullcalendar-event-3;
 }
 
-.fc-daygrid-day-events > :nth-child(4n+4) > .fc-daygrid-event > .fc-event-main > .fc-event-main-frame {
-  @apply bg-fullcalendar-event-4;
-}
-
 .fc-day {
-  @apply bg-white-ish !important;
+  @apply bg-white-ghost !important;
 
   &:hover {
-    @apply bg-blue-lavender !important; 
+    @apply bg-violet !important; 
 
     .fc-daygrid-day-number {
       @apply text-white !important;
     }
 
     .fc-col-header-cell-cushion, .fc-scrollgrid-sync-inner {
-      @apply bg-blue text-white !important;
+      @apply bg-violet text-white !important;
     }
   }
 }
 
 .fc-day-today {
-  @apply bg-blue-dark !important;
+  @apply bg-violet-russian !important;
 
   .fc-daygrid-day-number {
     @apply text-white !important;
   }
 }
 
 .fc-day-past {
-  @apply bg-gray-lighter !important;
+  @apply bg-white !important;
 }
 
 .fc-day-other {
-  @apply bg-gray-silver !important;
+  @apply bg-fullcalendar-event-3 !important;
 }
 
 .fc-col-header-cell > .fc-scrollgrid-sync-inner > * {
-  @apply text-gray-lighter font-normal;
+  @apply text-white font-normal;
 }
```

Comment:
There is a missing newline at the end of the document.

Reply:
Missing spaces added!